### PR TITLE
Add lemniscap gentx

### PR DESCRIPTION
**Pull request by zheli**
## Checklist

- [ ] New and updated code has appropriate documentation
- [ ] New and updated code has new and/or updated testing
- [ ] Required CI checks are passing
- [ ] Visual proof for any user facing features like CLI or documentation updates
- [ ] Linked issues closed with keywords

### CODE REVIEW

File: celestia/peers.txt [hunk 1]
```diff
@@ -8,3 +8,4 @@ f5bce79feec9068154e7f50fe8db7a8191a58663@82.220.38.204:10056
 d0c530290267e1c539f27061c9446d5042eaf1ca@161.97.180.20:26656
 a41b4bc451b1b71d537aa1018226e08d7fa7e44e@5.255.77.44:26656
 2f05b07cde80c5b08ee0159658f7977c7765f966@148.113.17.55:26756
+72dd61ecab64873e45c77f902fec73069dfb33f0@103.219.171.65:26656
```
Comment (evan-forbes) [inline celestia/peers.txt:11]: please submit a separate file in this director named lemniscap.json or simlply don't change this file

**Review by evan-forbes — CHANGES_REQUESTED** [review]

File: celestia/peers.txt (hunk 1 above)
Comment (evan-forbes) [reply celestia/peers.txt:11]: this is to avoid any conflicts in the peers.txt file.

Comment (zheli) [reply celestia/peers.txt]: :+1: will do.

**Comment by evan-forbes** [comment]
I went ahead and fixed the conflicts @zheli

**Review by evan-forbes — APPROVED** [review]
I went ahead and fixed the conflict with the peers.txt file

**Comment by zheli** [comment]
> I went ahead and fixed the conflict with the peers.txt file

Alright! Thx! I just saw your message :joy:

**Pull request closed without merging**